src/main/cpp/IntakeBehaviour.cpp

```diff
@@ -8,8 +8,10 @@ IntakeManualControl::IntakeManualControl(Intake *intake, frc::XboxController &co
 void IntakeManualControl::OnTick(units::second_t dt) {
   if (_codriver.GetBButtonPressed()) {
```

Comment:
this will only trigger once when you press it, you want GetBButton if you want to be able to hold it, but you should probably make this a toggle button